Description: PHPUnit 6 has moved to namespaced classes
Author: Nishanth Aravamudan <nish.aravamudan@canonical.com>
Bug: https://github.com/hamcrest/hamcrest-php/pull/47
Origin: upstream, https://github.com/hamcrest/hamcrest-php/commit/be5380f32221c57d4418617738108c5cac5ae78e
Forwarded: Will be done by Nishanth Aravamudan
Last-Update: 2018-02-15

--- a/tests/Hamcrest/AbstractMatcherTest.php
+++ b/tests/Hamcrest/AbstractMatcherTest.php
@@ -1,10 +1,12 @@
 <?php
 namespace Hamcrest;
 
+use \PHPUnit\Framework\TestCase;
+
 class UnknownType {
 }
 
-abstract class AbstractMatcherTest extends \PHPUnit_Framework_TestCase
+abstract class AbstractMatcherTest extends TestCase
 {
 
     const ARGUMENT_IGNORED = "ignored";
--- a/tests/Hamcrest/MatcherAssertTest.php
+++ b/tests/Hamcrest/MatcherAssertTest.php
@@ -1,7 +1,9 @@
 <?php
 namespace Hamcrest;
 
-class MatcherAssertTest extends \PhpUnit_Framework_TestCase
+use \PHPUnit\Framework\TestCase;
+
+class MatcherAssertTest extends TestCase
 {
 
     protected function setUp()
--- a/tests/Hamcrest/StringDescriptionTest.php
+++ b/tests/Hamcrest/StringDescriptionTest.php
@@ -1,6 +1,8 @@
 <?php
 namespace Hamcrest;
 
+use \PHPUnit\Framework\TestCase;
+
 class SampleSelfDescriber implements \Hamcrest\SelfDescribing
 {
     private $_text;
@@ -16,7 +18,7 @@
     }
 }
 
-class StringDescriptionTest extends \PhpUnit_Framework_TestCase
+class StringDescriptionTest extends TestCase
 {
 
     private $_description;
--- a/tests/Hamcrest/UtilTest.php
+++ b/tests/Hamcrest/UtilTest.php
@@ -1,7 +1,9 @@
 <?php
 namespace Hamcrest;
 
-class UtilTest extends \PhpUnit_Framework_TestCase
+use \PHPUnit\Framework\TestCase;
+
+class UtilTest extends TestCase
 {
 
     public function testWrapValueWithIsEqualLeavesMatchersUntouched()
